kani-compiler/src/codegen_cprover_gotoc/codegen/assert.rs

```diff
@@ -10,17 +10,19 @@ use cbmc::goto_program::{Expr, Location, Stmt};
 #[allow(dead_code)]
 pub enum PropertyClass {
     ArithmeticOverflow,
-    AssertFalse,
     Assume,
     Cover,
-    CustomProperty(String),
```

Comment:
Do we want to keep one property class that can be used as a "+" set member, to allow for edge case handling? I think `CustomProperty` could be useful for hotfixes which can be refactored later on. Not a blocker for merge though.

Reply:
I see your point. Do you have any use case in mind? I don't think fromStr() is used today.

My reasoning to remove this was:

1. Things could silently fail if we try to convert a string to a property class that doesn't exist.
2. This can be confusing for people adding properties in the future. If they want a new property class, should they use the `CustomProperty()` or add a new enum variant.

Reply:
Ah, I see your point. Thanks for the clarification!